index.html

```diff
@@ -325,9 +325,10 @@
       <dfn>register the observer</dfn> and must run these steps:</p>
       <ol data-link-for="PerformanceObserverInit">
         <li>Filter unsupported names within the `entryTypes` sequence, and
-        replace the `entryTypes` sequence with the new filtered sequence.</li>
-        <li>If the _options'_ <a>entryTypes</a> attribute is an empty sequence,
-        <a>throw</a> a JavaScript `TypeError`.
+        replace the `entryTypes` sequence with the new filtered sequence.
+        </li>
+        <li>If the resulting '<a>entryTypes</a>' sequence is an empty sequence, abort these steps.</li>
```

Comment:
Hmm, are we sure we want this here? As in, I think we still want the other steps to run?

Reply:
the observer will never get called since the list is empty. How would you differentiate from aborting the steps?

Reply:
On a second and more careful read.. you're right — nevermind, all good.